wiki/Help_centre/Tournament_bans/de.md

```diff
@@ -1,44 +1,40 @@
----
-no_native_review: true
----
-
 # Turniersperren
 
 *Hauptseite: [Hilfecenter](/wiki/Help_centre)*
 
 Eine Turniersperre verbietet es einem Spieler an offiziellen oder offiziell unterstützten Turnieren teilzunehmen. Inwieweit ein Spieler bei der Turnierorganisation mithelfen oder sich anderweitig bei diesen Turnieren einbringen kann, wird dadurch ebenfalls eingeschränkt.
 
-Turniersperren werden sehr selten ausgehändigt, aber im Allgemeinen unterliegen alle Spieler, die im Rahmen einer Standardbeschwerde für eine [Einschränkung des Accounts](/wiki/Help_centre/Account_restrictions) zurückkehren, einer Sperre von mindestens einem Jahr.
+Turniersperren werden sehr selten alleine ausgehändigt, aber in der Regel unterliegen alle Spieler, die im Rahmen einer Standardbeschwerde für eine [Einschränkung des Accounts](/wiki/Help_centre/Account_restrictions) zurückkehren, einer Sperre von mindestens einem Jahr.
 
 Durch die hohen Anforderungen bei Turnierspielen werden Turniersperren sehr ernst genommen und sind nicht Teil der Art von Nachsicht, die eine Accounteinschränkung üblicherweise erhält.
 
-## Was führt dazu, dass jemand eine vorübergehende Turniersperre erhält? {id=why-temporary}
+## Was führt dazu, dass jemand eine zeitige Turniersperre erhält? {id=why-temporary}
 
-Jeder schwerwiegende Verstoß der [Community-Regeln](/wiki/Rules), der während eines [offiziell unterstützten Turniers](/wiki/Tournaments/Official_support) stattfindet, wird wahrscheinlich eine Turniersperre verursachen. Auch dann, wenn es normalerweise nur zu einem Verstoß außerhalb der kompetitiven Spielumgebung geführt hätte.
+Jeder schwerwiegende Verstoß der [Community-Regeln](/wiki/Rules), der während eines [offiziell unterstützten Turniers](/wiki/Tournaments/Official_support) stattfindet, wird wahrscheinlich eine Turniersperre verursachen, auch wenn es normalerweise nur als einem kleineren Verstoß außerhalb der kompetitiven Spielumgebung gegolten hätte.
```

Comment:
```suggestion
Jede schwerwiegende Missachtung der [Community-Regeln](/wiki/Rules), die während eines [offiziell unterstützten Turniers](/wiki/Tournaments/Official_support) stattfindet, führt wahrscheinlich zu einer Turniersperre, auch wenn es sich nur um einen kleineren Verstoß außerhalb der kompetitiven Spielumgebung handelt.
```

Using [`Missachtung`](https://www.dwds.de/wb/missachten#d-1-2) here as a synonym for `Regelverstoß` to have more word variety.